meson.build

```diff
@@ -61,6 +61,12 @@ else
 endif
 conf.set10('HAVE_SPLIT_USR', split_usr)
 
+if get_option('split-bin') == 'auto'
+        split_bin = run_command('test', '-L', '/usr/sbin').returncode() != 0
+else
+        split_bin = get_option('split-bin') == 'true'
+endif
```

Comment:
i figure sooner or later it would have sense to also define HAVE_SPLIT_BIN or so as c macro, so that we can adjust search paths depending on thus, and drop sbin where it is redundant.